src/lwc/gsChecklistItem/gsChecklistItem.html

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="step-card">
+        <div class="card-image">
+            <img if:true={hasLink} src={imgSrc}/>
+        </div>
+        <div class="card-content">
+            <p class="content-time">{item.extraInfo}</p>
+            <h1 class="content-title">{item.title}</h1>
+            <div  class="content-text">
+                {item.description}
+            </div>
+        </div>
+        <div class="content-footer">
+            <div class="content-footer-aux">
+                <a if:true={hasLink} href={item.link.href} target="_blank">
+                    {item.link.label}
+                </a>
+                <div if:false={hasLink}></div>
+                <div class="footer-buttons">
+                    <button if:true={hasSecondaryBtn} 
+                        onclick={onClickSecondaryBtn}
```

Comment:
How do these onclick handlers announce to a screen reader?

Reply:
@bethbrains Did you get an answer to this question?

Reply:
@bethbrains Yes is read for the screen reader